fedmsg_meta_fedora_infrastructure/scm.py

```diff
@@ -193,6 +202,12 @@ def usernames(self, msg, **config):
             return set()
 
     def packages(self, msg, **config):
+
+        # If its not an rpms/ thing, then its not a package.
+        if 'namespace' in msg['msg']:
+            if msg['msg']['namespace'] != 'rpms':
+                return set()
+
```

Comment:
Should we have some specific notifications/conversion for the ``*-checks`` repo? I figure some people would like to be notified of changes done in them.